Cargo.toml

```diff
@@ -43,6 +43,9 @@ smart-default = "0.6"
 swayipc-async = "2.0"
 tokio-stream = "0.1"
 toml = "0.7"
+chrono = "0.4"
```

Comment:
@virajbhartiya , it's unhappy that you have `chrono` and `chrono-tz` here becuase they are further down:

```toml
[dependencies.chrono]
version = "0.4"
default-features = false
features = ["clock", "unstable-locales"]

[dependencies.chrono-tz]
version = "0.8"
features = ["serde"]
```